services/dialog/v1.js

```diff
@@ -146,31 +146,31 @@ module.exports = function (RED) {
           params.dialog_id = dialogid;
           params.input = msg.payload;
 		  
-          if (config.mode === "startconverse" || config.mode === "converse") {			  
-            node.status({fill:"blue", shape:"dot", text:"Starting Dialog Conversation"});
+          if (config.mode === 'startconverse' || config.mode === 'converse') {			  
+            node.status({fill:'blue', shape:'dot', text:'Starting Dialog Conversation'});
             dialog.conversation (params, function (err, dialog_data) {
+              node.status({});
               if (err) {
-                node.status({fill:"red", shape:"ring", text:"call to dialog service failed"}); 
-                console.log("Error:", msg, err);
+                node.status({fill:'red', shape:'ring', text:'call to dialog service failed'}); 
+                console.log('Error:', msg, err);
                 node.error(err, msg);
               } else {
-                node.status({fill:"green", shape:"dot", text:"dialog conversation successful"});		  
                 msg.dialog = dialog_data;		  
-                msg.payload = "Check msg.dialog dialog data";
+                msg.payload = 'Check msg.dialog dialog data';
                 node.send(msg);
               }   
             });
           }
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Closing curly brace does not appear on the same line as the subsequent block.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2462016486/issues/source?bid=3225115&fileBranchId=3255594#l164)